lib/c-api/tests/wasm-c-api/example/multi.c

```diff
@@ -7,6 +7,19 @@
 
 #define own
 
+int wasmer_last_error_length(void);
+void wasmer_last_error_message(char*, int);
+
+// Use the last_error API to retrieve error messages
+void print_wasmer_error()
+{
+  int error_len = wasmer_last_error_length();
+  printf("Error len: `%d`\n", error_len);
+  char *error_str = malloc(error_len);
+  wasmer_last_error_message(error_str, error_len);
+  printf("Error str: `%s`\n", error_str);
+}
+
```

Comment:
That's annoying for further updates of the `wasm-c-api` repo :-/. But I don't see how to solve this.

Reply:
Oops, that was debug code... I didn't intend to ship that -- thanks for pointing it out!